lib/osf-components/addon/components/sort-button/template.hbs

```diff
@@ -1,18 +1,16 @@
 <Button
-    data-test-ascending-sort={{this.sortBy}}
-    data-analytics-name={{concat 'Ascending sort ' this.sortBy}}
-    local-class={{if (eq @sort this.sortBy) 'selected' 'not-selected'}}
-    title={{t 'general.sort_asc'}}
+    data-test-sort-button={{this.sortBy}}
+    data-analytics-name={{concat 'Sort ' this.sortBy}}
+    local-class='sort-button'
```

Comment:
Re: `local-class`, our CSS uses what is called [ember-css-modules](https://github.com/salsify/ember-css-modules), so any new class name in this `local-class=` statement should be added to the corresponding template's `styles.scss` file. If there's changes to the scss file for this component, don't forget to add those changes to this PR

Re: indicating active sort method, I've checked this out locally, and it seems there's not a good way to tell which sort is the active option aside from when it shows the upward arrow. Can we add back the existing `{{if (eq @sort this.sortBy) 'selected' 'not-selected'}}` bit to make it clearer when this is the selected sort option?